mlrun/api/api/endpoints/functions.py

```diff
@@ -248,12 +248,20 @@ async def start_function(
         mlrun.api.schemas.AuthorizationAction.update,
         auth_info,
     )
+    background_timeout = mlrun.mlconf.background_tasks_timeout_defaults.runtimes.dask
+    if mlrun.mlconf.background_tasks_timeout_defaults.mode == "enabled":
+        background_timeout = (
+            int(background_timeout) if background_timeout is not None else None
+        )
 
     background_task = await run_in_threadpool(
-        mlrun.api.utils.background_tasks.Handler().create_project_background_task,
-        function.metadata.project,
+        mlrun.api.utils.background_tasks.Handler().create_background_task,
+        db_session,
         background_tasks,
         _start_function,
+        function.metadata.project,
+        background_timeout,
+        # args for _start_function
```

Comment:
Change args order so that the background task function and it's args/kwargs will be the last ones

Reply:
I now understand you did it because of the `None` defaults, nvm

Reply:
Project can be returned to be the first argument

Reply:
it feels like we always pass the db_session first, so moved project to be the second argument.